### PR TITLE
Bump build dependency `cfg_aliases` to 0.2

### PR DESCRIPTION
**Pull request by waywardmonkeys**
- [x] Tested on all platforms changed
- [ ] Added an entry to `CHANGELOG.md` if knowledge of this change could be valuable to users
- [ ] Updated documentation to reflect any user-facing changes, including notes of platform-specific behavior
- [ ] Created or updated an example program if it would help users understand this functionality
- [ ] Updated [feature matrix](https://github.com/rust-windowing/winit/blob/master/FEATURES.md), if new features were added or implemented

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -58,7 +58,7 @@ rwh_05 = ["dep:rwh_05", "ndk/rwh_05"]
 rwh_06 = ["dep:rwh_06", "ndk/rwh_06"]
 
 [build-dependencies]
-cfg_aliases = "0.1.1"
+cfg_aliases = "0.2"
```
Comment (kchibisov) [inline Cargo.toml:61]:
```suggestion
cfg_aliases = "0.2.0"
```

**Review by kchibisov — CHANGES_REQUESTED** [review]
Always specify full latest version available at the time of submitting the patch.

**Comment by waywardmonkeys** [comment]
@kchibisov Updated!

**Comment by kchibisov** [comment]
Thanks.

**Pull request merged**